Cargo.toml

```diff
@@ -3,46 +3,53 @@ resolver = "2"
 members = ["crates/*"]
 
 [workspace.dependencies]
-anyhow = "1"
-async-trait = "0.1"
+anyhow = "1.0.82"
+async-trait = "0.1.80"
+axum = "0.6"
 bigdecimal = "0.3"
-cached = { version = "0.44", default-features = false }
-chrono = { version = "0.4", default-features = false }
-clap = { version = "4", features = ["derive", "env"] }
-derivative = "2"
-ethcontract = { version = "0.25.4", default-features = false, features = ["aws-kms"] }
+cached = { version = "0.49.3", default-features = false }
+chrono = { version = "0.4.38", default-features = false }
+clap = { version = "4.5.4", features = ["derive", "env"] }
+derivative = "2.2.0"
+ethcontract = { version = "0.25.5", default-features = false, features = ["aws-kms"] }
 ethcontract-generate = { version = "0.25", default-features = false }
 ethcontract-mock = { version = "0.25", default-features = false }
-ethereum-types = "0.14"
-futures = "0.3"
+ethereum-types = "0.14.1"
+flate2 = "1.0.28"
+futures = "0.3.30"
 gas-estimation = { git = "https://github.com/cowprotocol/gas-estimation", tag = "v0.7.3", features = ["web3_", "tokio_"] }
-hex = { version = "0.4", default-features = false }
-hex-literal = "0.4"
+hex = { version = "0.4.3", default-features = false }
+hex-literal = "0.4.1"
 humantime = "2.1.0"
 humantime-serde = "1.1.1"
+hyper = "0.14.28"
 indexmap = "2.2.5"
-itertools = "0.11"
-lazy_static = "1"
-maplit = "1"
-mockall = "0.11"
-num = "0.4"
-once_cell = "1"
+itertools = "0.12.1"
+lazy_static = "1.4.0"
+maplit = "1.0.2"
+mockall = "0.12.1"
+num = "0.4.2"
+once_cell = "1.19.0"
 primitive-types = "0.12"
-prometheus = "0.13"
+prometheus = "0.13.3"
 prometheus-metric-storage = "0.5.0"
-rand = "0.8"
-regex = "1"
-reqwest = "0.11"
-secp256k1 = "0.27"
-serde = { version = "1", features = ["derive"] }
-serde_json = "1"
-serde_with = "3"
+rand = "0.8.5"
+regex = "1.10.4"
+reqwest = "0.11.27"
+secp256k1 = "0.27.0"
```

Comment:
We cannot update to `0.29.0` until `web3` crate is not updated to the aforementioned version.

Reply:
The PR for the crate `ethceontract-rs` : https://github.com/cowprotocol/ethcontract-rs/pull/969